metashare/api/models.py

```diff
@@ -397,7 +397,9 @@ def get_head(self):
 
     def should_update_in_progress(self):
         task_statuses = self.tasks.values_list("status", flat=True)
-        return any(status == TASK_STATUSES["In progress"] for status in task_statuses)
+        return task_statuses and not all(
+            status == TASK_STATUSES.Planned for status in task_statuses
+        )
```

Comment:
@wlonk I realized that before, a project with one Completed task and another Planned task was not correctly showing as In Progress. So I made this change and the corresponding reordering of the `update_status` `if` below. Feel free to cleanup -- if you don't like relying on the specific ordering of the `if / elif`, then we could add a `not self.should_update_review()` here.